pkg/rewards/show.go

```diff
@@ -223,18 +230,19 @@ func handleRewardsOutput(
 		}
 		fmt.Println()
 		fmt.Println(strings.Repeat("-", 30), msg, strings.Repeat("-", 30))
-		printRewards(rewards)
+		printRewards(allRewards)
 	}
 	return nil
 }
 
-func printRewards(rewards map[gethcommon.Address]*big.Int) {
+func printRewards(allRewards allRewardsJson) {
 	// Define column headers and widths
 	headers := []string{
+		"Token Name",
 		"Token Address",
 		"Amount (Wei)",
 	}
-	widths := []int{46, 30}
+	widths := []int{20, 46, 30}
```

Comment:
nits: nice to have these as constants to be more self-explanatory of their meaning.

Reply:
they are just the width of a columns, I can add that comment but those numbers are arbitrary and doesn't have any significance.

Reply:
Totally get that, but leaving them unnamed makes them "magic numbers". 

[Magic numbers stackoverflow](https://stackoverflow.com/questions/47882/what-are-magic-numbers-and-why-do-some-consider-them-bad)

Reply:
I guess the only readability improvement it makes is you can know it's tokenname width which is fair.

Reply:
Yeah, I usually really like it because it immediately hints at contextual reasoning. For example, if it's a token address, familiarity with max address length. It can be inferred from reading but I find this helps me read smoother